pkg/network/render.go

```diff
@@ -403,10 +396,7 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	// Validate that this is either a BareMetal or None PlatformType. For all other
 	// PlatformTypes, migration to DualStack is prohibited
```

Comment:
We should probably update this comment too (or remove it) since Vsphere also support dualstack conversion.